### PR TITLE
Analyze trace in Visual Studio

### PR DESCRIPTION
**Pull request by mikelle-rogers**
Adding that trace files can be opened in Visual Studio. Added a link to a VS CPU analysis tutorial.

### CODE REVIEW

File: docs/core/diagnostics/dotnet-trace.md [hunk 1]
```diff
@@ -405,7 +405,7 @@ However, when you want to gain a finer control over the lifetime of the app bein
 
 ## View the trace captured from dotnet-trace
 
-On Windows, *.nettrace* files can be viewed on [PerfView](https://github.com/microsoft/perfview) for analysis: For traces collected on other platforms, the trace file can be moved to a Windows machine to be viewed on PerfView.
+On Windows, *.nettrace* files can be viewed on [Visual Studio](https://docs.microsoft.com/en-us/visualstudio/profiling/beginners-guide-to-performance-profiling?view=vs-2022#step-2-analyze-cpu-usage-data) or [PerfView](https://github.com/microsoft/perfview) for analysis: For traces collected on other platforms, the trace file can be moved to a Windows machine to be viewed on Visual Studio or PerfView.
```
Comment (gewarren) [inline docs/core/diagnostics/dotnet-trace.md:408]:
```suggestion
On Windows, you can view *.nettrace* files in [Visual Studio](/visualstudio/profiling/beginners-guide-to-performance-profiling#step-2-analyze-cpu-usage-data) or [PerfView](https://github.com/microsoft/perfview) for analysis.
```

File: docs/core/diagnostics/dotnet-trace.md [hunk 2]
```diff
@@ -405,7 +405,7 @@ However, when you want to gain a finer control over the lifetime of the app bein
...
 
 On Linux, the trace can be viewed by changing the output format of `dotnet-trace` to `speedscope`. The output file format can be changed using the `-f|--format` option - `-f speedscope` will make `dotnet-trace` produce a `speedscope` file. You can choose between `nettrace` (the default option) and `speedscope`. `Speedscope` files can be opened at <https://www.speedscope.app>.
```
Comment (gewarren) [inline docs/core/diagnostics/dotnet-trace.md:410]:
```suggestion
On Linux, you can view the trace by changing the output format of `dotnet-trace` to `speedscope`. Cange the output file format by using the `-f|--format` option. You can choose between `nettrace` (the default option) and `speedscope`. The option `-f speedscope` will make `dotnet-trace` produce a `speedscope` file. `Speedscope` files can be opened at <https://www.speedscope.app>.

For traces collected on non-Windows platforms, you can also move the trace file to a Windows machine to view it in Visual Studio or PerfView.
```

**Review by gewarren — COMMENTED** [review]
Left a few suggestions.

File: docs/core/diagnostics/dotnet-trace.md (hunk 1 above)
Comment (mikelle-rogers) [reply docs/core/diagnostics/dotnet-trace.md:408]: Thank you for your suggestions! I think it is important to leave in the last sentence for traces collected on other platforms.

**Review by gewarren — APPROVED** [review]
Thanks!

**Pull request merged**